cmake/common/px4_base.cmake

```diff
@@ -507,7 +506,11 @@ function(px4_add_common_flags)
 		#               but generates too many false positives
 		)
 
-	if (NOT ${CMAKE_C_COMPILER_ID} STREQUAL "Clang")
+	if (${CMAKE_C_COMPILER_ID} STREQUAL "Clang")
```

Comment:
This is the bizzaro cmake syntax for else. if it was an elseif you would have a different condition, but an else replicates the if condition above.